samples/viewer/Markers with size uncertain data.html

```diff
@@ -45,9 +45,10 @@
                     else if (headers[j] == 'w_ub95') size_u95.push(parseFloat(data1[j]));
                 }
             }
+            x.pop();
             ChartViewer.show($("#chart"), {
                 "markers": Plot.markers({
-                    x: x, y: y, color: color, size: { median: size_mean, lower95: size_l95, upper95: size_u95, lower68: size_l68, upper68: size_u68 },
+                    x: x, y: y, shape: 'petals', color: color, size: { median: '15.0', lower95: size_l95, upper95: size_u95, lower68: size_l68, upper68: size_u68 },
```

Comment:
@ElenaPochernina Please fix the issues and **document** these changes:

1. Shape `petals` should expect only 95 or 68 quantiles because it uses either of them actually.
2. Property `size.median` for `petals` is not actually used for rendering; instead the shape needs size in pixels since all markers are of same size in pixels. I propose to use property `diameter` which MUST be a number:

```
... size: { diameter: 15.0, lower95: size_l95, upper95: size_u95 }, ...
```